### DISCUSSION TITLE
3.0.1 version

### DISCUSSION THREAD

**Discussion by tonylu110** [tonylu110/uyou-todo-electron#41] [Announcements]
- Fixed Windows 10 use
- Fixed setup page images

<hr /><em>This discussion was created from the release <a href='https://github.com/tonylu110/uyou-todo-electron/releases/tag/3.0.1'>3.0.1 version</a>.</em>

**Comment by I-icrous-I** [comment]
目前遇到以下情况，在更新3.0.1版本后，在初次启动时，选择了便利贴模式，现在无法退出该模式。
关闭软件重新运行、重新启动电脑，依旧运行于便利贴模式。
界面中，无法通过右键进入设置模式。
同时无法查看已经建立的其他任务。
需要协助，是否需要重装，数据是否会丢失？
![uyou_ToDo_wmWu9QXS4Z](https://github.com/user-attachments/assets/0d85ed52-b0c1-4260-89fb-125b6432132c)
-c6ddf0c433de)

**Reply by I-icrous-I** [reply]
![Screenshot_2024-07-25-11-22-12-53_e39d2c7de19156b](https://github.com/user-attachments/assets/e569f6ed-b3f7-44d3-b8a4-66b26ac2c325)

**Reply by I-icrous-I** [reply]
另外催一下手机端

**Reply by tonylu110** [reply]
在登录的情况下数据是同步的，登录账号同步即可，重新安装一下吧

**Reply by tonylu110** [reply]
可以安装一下之前的老版本回退一下

**Reply by tonylu110** [reply]
或者稍等一下3.0.2的版本更新

**Comment by tonylu110** [comment]
[3.0.2](https://github.com/tonylu110/uyou-todo-electron/releases/tag/3.0.2)

**Reply by I-icrous-I** [reply]
覆盖安装后正常了。
